tests/operations/tests/HaversineDistance.mjs

```diff
@@ -11,7 +11,18 @@ TestRegister.addTests([
     {
         name: "Haversine distance",
         input: "51.487263,-0.124323, 38.9517,-77.1467",
-        expectedOutput: "5619355.701829259",
+        expectedOutput: "5902.542836307819",
```

Comment:
When I run this recipe I get the output `5902542.836307819`, could you double check the answer here is what you get? I think maybe this has been converted to kilometers when the default output is meters.

Reply:
You are correct. I'll amend. Thanks @mt3571